### ISSUE TITLE
Click handler fires twice on the mobile device when the 'detect' is set to BOTH

### ISSUE THREAD

**Issue by akosarch** [minwork/use-long-press#3]
### Steps to recreate:
- Open the [codesandbox](https://codesandbox.io/embed/stupefied-grass-jgwuv) on your desktop;
- Open the console and click on the button. Handler fires only once per click, as expected;
- Then open the Chrome's device emulator -> pick a mobile device (eg iPhone);
- Click on the button again. Now you should see in the console that the handler fires twice per click;

**Note:** when 'detect' is set to 'MOUSE' or 'TOUCH' it works as expected; The bug occurs only when 'BOTH' is set.
Video with the bug captured: https://www.dropbox.com/t/PEvlUSmBtV2hWlYc

**Comment by minwork** [comment]
Hello,

thanks for the report and an example illustrating that bug.

I will investigate it.

**Comment by minwork** [comment]
@akosarch Fixed in v1.0.5

**Comment by chriswardo** [comment]
Hi! It seems like the onCancel (which I want to use as the onClick) event fires twice when just tapping using Chrome's device emulator (and the iOS Simulator).

**Comment by minwork** [comment]
Hey @chriswardo I will check it. In the meantime could you create separate issue for this case (preferably with reproducible example)?
